test/controllers/home.js

```diff
@@ -0,0 +1,29 @@
+const supertest = require('supertest');
+const tape = require('tape');
+const app = require('../../src/app');
+
+
+tape('testing the home route content type and status code', (t) => {
+  supertest(app)
+    .get('/home')
+    .expect(200)
+    .expect('Content-Type', 'text/html; charset=utf-8')
+    .end((err, res) => {
+      t.error(err);
```

Comment:
```node
if (err) return t.error(err);
```
If an `err` exists, exit with an error, do not continue the execution and do not create an error if there isn't an error.